### PR TITLE
Add v15.0.0 for azure

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

File: azure/v15.0.0/README.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
+# :zap: Giant Swarm Release v15.0.0 for Azure :zap:
+
+This is the first Giant Swarm Azure release featuring Kubernetes 1.20.
```
Comment (whites11) [inline azure/v15.0.0/README.md:3]:
```suggestion
This is the first Giant Swarm Azure stable release featuring Kubernetes 1.20.
```

File: azure/v15.0.0/release.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    version: 0.27.1
+  - name: kubernetes
+    version: 1.20.6
+  - name: containerlinux
+    version: 2605.12.0
+  - name: calico
+    version: 3.15.3
+  - name: etcd
+    version: 3.4.14
+  date: "2021-05-24T11:40:38Z"
```
Comment (whites11) [inline azure/v15.0.0/release.yaml:54]: Bump the date please

File: azure/v15.0.0/release.diff [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    version: 0.27.1                                                    version: 0.27.1
+  - name: kubernetes                                                 - name: kubernetes
+    version: 1.20.6                                                    version: 1.20.6
+  - name: containerlinux                                             - name: containerlinux
+    version: 2605.12.0                                                 version: 2605.12.0
+  - name: calico                                                     - name: calico
+    version: 3.15.3                                                    version: 3.15.3
+  - name: etcd                                                       - name: etcd
+    version: 3.4.14                                                    version: 3.4.14
+  date: "2021-05-14T13:55:02Z"                                  |    date: "2021-05-24T11:40:38Z"
```
Comment (whites11) [inline azure/v15.0.0/release.diff:54]: Bump the date please

File: azure/v15.0.0/release.yaml (hunk 2 above)
Comment (fiunchinho) [reply azure/v15.0.0/release.yaml]: Done

File: azure/v15.0.0/release.diff (hunk 3 above)
Comment (fiunchinho) [reply azure/v15.0.0/release.diff]: Done

File: azure/v15.0.0/README.md (hunk 1 above)
Comment (fiunchinho) [reply azure/v15.0.0/README.md]: Done

File: azure/v15.0.0/release.diff [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    version: 0.27.1                                                    version: 0.27.1
+  - name: kubernetes                                                 - name: kubernetes
+    version: 1.20.6                                                    version: 1.20.6
+  - name: containerlinux                                             - name: containerlinux
+    version: 2605.12.0                                                 version: 2605.12.0
+  - name: calico                                                     - name: calico
+    version: 3.15.3                                                    version: 3.15.3
+  - name: etcd                                                       - name: etcd
+    version: 3.4.14                                                    version: 3.4.14
+  date: "2021-06-14T11:40:38Z"                                  |    date: "2021-05-24T11:40:38Z"
```
Comment (whites11) [inline azure/v15.0.0/release.diff:54]: I think this is wrong. The left side is the previous release, the right side is the new one. You should have bumped the right side date

**Review by whites11 — APPROVED** [review]
LET'S DO IT

**Pull request merged**